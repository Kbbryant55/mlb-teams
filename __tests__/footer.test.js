import Footer from "../src/components/footer";
import { render, screen } from "@testing-library/react";
import "@testing-library/jest-dom";

describe("Footer", () => {
  it("renders a footer", () => {
    render(<Footer copyrightLink="" linkLabel="" copyrightText="" />);

    expect(screen.getByTestId("footer")).toBeInTheDocument();
  });

  it("renders Footer unchanged", () => {
    const { container } = render(<Footer />);
    expect(container).toMatchSnapshot();
  });
});
