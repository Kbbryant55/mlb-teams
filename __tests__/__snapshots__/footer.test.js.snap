// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders Footer unchanged 1`] = `
<div>
  <p
    class="mt-20 text-center"
    data-testid="footer"
  >
    <a
      class="text-blue-500 hover:underline"
      target="_blank"
    >
      Â 
      .
    </a>
  </p>
</div>
`;
